### PR TITLE
new: add signature on automations

### PR DESCRIPTION
**Pull request by t00f**
Adding signature to automations

### CODE REVIEW

File: specs/@signed.abs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Attributes
+attributes:
+  v1:
+  - name: signature
+    description: Signature to validate the authenticity of the object.
+    type: string
+    exposed: true
```
Comment (primalmotion) [inline specs/@signed.abs:7]: add a getter/setter here. will help make this completely generic in bahamut

Comment (t00f) [reply specs/@signed.abs:7]: Done

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "5b2dab8a51911143172796143e82e74834c6c1fe",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "9cc432ebbebacdc5b883c227c7594c09f2e4ce56",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "5301617ca816b41009ac7196844b49c35881f501",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "5301617ca816b41009ac7196844b49c35881f501",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "5b2dab8a51911143172796143e82e74834c6c1fe",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "9cc432ebbebacdc5b883c227c7594c09f2e4ce56",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "5301617ca816b41009ac7196844b49c35881f501",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "5b2dab8a51911143172796143e82e74834c6c1fe",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "9cc432ebbebacdc5b883c227c7594c09f2e4ce56",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "9fd3e1d932ed932a4507de579c4bbbd76eb55d7e",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "5b2dab8a51911143172796143e82e74834c6c1fe",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "9cc432ebbebacdc5b883c227c7594c09f2e4ce56",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "c500d5ae8f8d34e396a5345eff03513a5321c8bb",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "5b2dab8a51911143172796143e82e74834c6c1fe",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "9cc432ebbebacdc5b883c227c7594c09f2e4ce56",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "3ad0af651f68f73ade7efc6fdecce2170147d802",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "0f79a899e0fed6042ac0fb3a3a08b83a5920ceb7",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "212b809e313ca29a30d2a87861cf285ff08392ad",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "212b809e313ca29a30d2a87861cf285ff08392ad",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "3ad0af651f68f73ade7efc6fdecce2170147d802",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "0f79a899e0fed6042ac0fb3a3a08b83a5920ceb7",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "0f79a899e0fed6042ac0fb3a3a08b83a5920ceb7",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "212b809e313ca29a30d2a87861cf285ff08392ad",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "2830d8b8f059d8294c0162bb9ee45e7ae84a388b",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "0f79a899e0fed6042ac0fb3a3a08b83a5920ceb7",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "212b809e313ca29a30d2a87861cf285ff08392ad",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "dd4c2e034e7129960e02cf29f11acd8f25bb6047",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "f148bcc135a431dd0294b4e731c99416c7a7ddde",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "0f79a899e0fed6042ac0fb3a3a08b83a5920ceb7",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "212b809e313ca29a30d2a87861cf285ff08392ad",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automations-july",
    "component": "gaia",
    "pr-id": "519",
    "commit-sha": "0f79a899e0fed6042ac0fb3a3a08b83a5920ceb7",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "underwater",
    "pr-id": "115",
    "commit-sha": "212b809e313ca29a30d2a87861cf285ff08392ad",
    "pipeline": "master"
  },
  {
    "project": "automations-july",
    "component": "backend",
    "pr-id": "786",
    "commit-sha": "f148bcc135a431dd0294b4e731c99416c7a7ddde",
    "pipeline": "master"
  }
]
```

**Pull request merged**